src/main/java/org/redisson/RedissonListMultimap.java

```diff
@@ -65,6 +65,19 @@
                 "return size; ",
                 Arrays.<Object>asList(getName()));
     }
+    
+    public Future<Integer> keySizeAsync() {
+    	return commandExecutor.evalReadAsync(getName(), codec, RedisCommands.EVAL_INTEGER,
```

Comment:
Please replace it to `return commandExecutor.readAsync(getName(), LongCodec.INSTANCE, RedisCommands.HLEN, getName());` instead due to performance gain